pypeit/spectrographs/spectrograph.py

```diff
@@ -1146,7 +1146,12 @@ def get_rawimage(self, raw_file, det):
         """
         # Check extension and then open
         self._check_extensions(raw_file)
-        hdu = io.fits_open(raw_file)
+        try: 
+            hdu = io.fits_open(raw_file)
+        except Exception as e:
```

Comment:
Is there a more specific exception type that we can catch related to the end card missing?

Reply:
It is the same OSError as you have in io.py so I changed to used that here now.